src/main/java/org/example/learningsystem/core/template/service/MustacheTemplateService.java

```diff
@@ -0,0 +1,37 @@
+package org.example.learningsystem.core.template.service;
+
+import com.samskivert.mustache.Mustache;
+import lombok.extern.slf4j.Slf4j;
+import org.example.learningsystem.core.template.exception.RenderTemplateException;
+import org.springframework.boot.autoconfigure.mustache.MustacheResourceTemplateLoader;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class MustacheTemplateService implements RenderTemplateService {
+
+    private static final String TEMPLATES_PREFIX = "templates/";
+    private static final String TEMPLATES_SUFFIX = ".mustache";
```

Comment:
these veriables could be defined via env vars (spring.mustache.suffix and .postfix). Then you can just inject `MustacheResourceTemplateLoader` prepared by spring instead of creating it manually